plugins/catalog/src/alpha/pages.tsx

```diff
@@ -100,7 +108,7 @@ export const catalogEntityPage = PageBlueprint.makeWithOverrides({
 
         const header = inputs.header?.get(
           EntityHeaderBlueprint.dataRefs.element,
-        ) ?? <EntityHeader contextMenuItems={menuItems} />;
+        );
```

Comment:
I believe this change is safe to do since the alpha `<EntityLayout />` falls back to this same header and passes on its `contextMenuItems` prop.